components/support/sync15-traits/src/bridged_engine.rs

```diff
@@ -153,14 +158,22 @@ impl IncomingEnvelope {
 pub struct OutgoingEnvelope {
     id: Guid,
     cleartext: String,
+    sortindex: Option<i32>,
+    ttl: Option<u32>,
 }
 
 impl From<Payload> for OutgoingEnvelope {
     fn from(payload: Payload) -> Self {
+        let mut payload = payload;
```

Comment:
```suggestion
    fn from(mut payload: Payload) -> Self {
```

I think we can declare the binding as `mut` in the argument, and avoid shadowing—though this also works!

Reply:
Ah yeah that makes sense, will commit this!